app/Services/ImportResults.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Services;
+
+use App\Models\ImportMeta;
+use App\Models\Mismatch;
+
+class ImportResults
+{
+    public function generateCSV(string $path, ImportMeta $import): void
+    {
+        $mismatches = Mismatch::get();
+
+        // Creating the output stream
+        $handle = fopen($path, 'w');
+
+        // Get the mismatches as a LazyCollection
+        $mismatches = Mismatch::whereBelongsTo($import)->cursor();
```

Comment:
Thanks for using a lazy collection, it's a good idea. Also, the comment makes it explicit so that we know which docs to refer to.